tools/diagnosticsHelpers/displayTopology.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+import networkx as nx
+import scipy
+import os
+import csv
+import argparse
+
+parser = argparse.ArgumentParser()
+   
+parser.add_argument('--file', type=str, required=False, help='Path to the network topology - .csv file')
+
+args = parser.parse_args()
+
+if (args.file is None):
+    
+    #We get file from expect path, relative to this one
+    sub_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
don't try to find the absolute path, by default use only the name of the file expecting it to be in the current directory and display a message in case the file not exist.

It's better to keep the code simple than to try to guess the needs of the user.